mesa_geo/geoagent.py

```diff
@@ -195,9 +179,9 @@ def from_GeoDataFrame(self, gdf, unique_id="index", set_attributes=True):
                 )
 
         agents = []
-        for index, row in gdf.iterrows():
+        for _index, row in gdf.iterrows():
```

Comment:
since you don't use `_index` in the loop, the python convention I thought is to use `_`

Reply:
I think we even can do `for row in gdf.itertuples():`